src/github.com/matrix-org/dendrite/roomserver/storage/events_table.go

```diff
@@ -297,6 +295,28 @@ func (s *eventStatements) bulkSelectEventReference(eventNIDs []types.EventNID) (
 	return results, nil
 }
 
+func (s *eventStatements) bulkSelectEventID(eventNIDs []types.EventNID) (map[types.EventNID]string, error) {
```

Comment:
Comment the value of the output map please. I assume this is `eventNID -> event_id`.